cnd/src/seed.rs

```diff
@@ -85,4 +233,60 @@ mod tests {
         let debug = format!("{:?}", seed);
         assert_eq!(debug, "Seed([*****])".to_string());
     }
+
+    // 32 bytes base64 encoded.
+    const PEM: &str = "-----BEGIN SECRET SEED-----
+syl9wSYaruvgxg9P5Q1qkZaq5YkM6GvXkxe+VYrL/XM=
+-----END SECRET SEED-----
+";
+
+    #[test]
+    fn pem_library_works_as_expected() {
```

Comment:
:question: Is this test necessary, since you are only testing that a hopefully already tested library works as advertised?

Reply:
Removed, thanks.